### DISCUSSION TITLE
Wikifiles not accurate?

### DISCUSSION THREAD

**Discussion by HikeAndMap** [emgucv/emgucv#570] [Q&A] [answered]
https://www.emgu.com/wiki/files/4.5.3/document/html/ac52a5f1-d607-5509-50df-885e3268a455.htm

Emgu.CV.BitmapExtension
public static class BitmapExtension

there's no Emgu.CV.BitmapExtension -where is it? How to use it?

**Comment by emgucv** [comment]
If you are using nuget package, the namespace is included in the Emgu.CV.Bitmap nuget package:
https://www.nuget.org/packages/Emgu.CV.Bitmap

**Reply by HikeAndMap** [reply]
I just compiled the whole package myself and use the cvextern.dll
In there everything seems to be working - except the bitmap part...

thanks for clearing the nuget works though

**Comment by emgucv** [comment, marked as answer]
The source code for the Bitmap extension project is available here:
https://github.com/emgucv/emgucv/tree/master/Emgu.CV.Platform/Emgu.CV.Bitmap

The "Emgu.CV.Bitmap" project is part of the Emgu.CV.Sln:
https://github.com/emgucv/emgucv/blob/master/Solution/Windows.Desktop/Emgu.CV.sln

**Reply by HikeAndMap** [reply]
Thank you!
